profile/README.md

```diff
@@ -1,7 +1,34 @@
-## OpenHW Group
+<img src="https://www.openhwgroup.org/images/openhw-landscape.png" width="418px" height="103px" /> <img src="https://www.openhwgroup.org/images/core-v-portrait.png" align="right" width="150px" height="120px"/>
 
+# About OpenHW Group
+
+OpenHW Group is a not-for-profit, global organization driven by its members and individual contributors where hardware and software designers collaborate in the development of open-source cores, related IP, tools and software. OpenHW provides an infrastructure for hosting high quality open-source HW developments in line with industry best practices.
```

Comment:
Recently one Member commented about OpenHW developing open source _tools_.  For example, one of the Big3 EDA vendors might be unhappy if we were to start working on an SV simulator.  Of course, we are not doing that, but perhaps we should drop the word _tools_ from this statement just to avoid confusion.

Also, I would not be a very good Director of Engineering for the _Verification_ Task Group if I did not advocate for an explicit mention of verification IP.   So how about something like: `...hardware and software designers collaborate in the development of open-source cores, related design and verification IP and software.`

Reply:
I do not see the word `tools` as a problem but I see your point. Although `tools` is a very ample word, we also do open-source tools so probably we should not drop it.

As @MikeOpenHWGroup said probably I would mention the `verification` word somewhere :+1:

Reply:
this work @DBees has done is about packaging up our existing positioning into an intro section for our repos - let's not edit / change positioning text at the same time.  We can do so as needed and that text will get re-used in several places...

Reply:
Correct @rickoco this text is a straight cut and paste from the website text. The point here is to agree on what we want the landing section to look like. I'll continue to add other project repos in the ToC section.